cli/replicaset/configsource.go

```diff
@@ -119,8 +119,7 @@ func (c *CConfigSource) patchInstanceConfig(instanceName string, force bool,
 func (c *CConfigSource) patchConfigWithRoles(ctx RolesAddCtx,
 	getPathFunc func(clusterConfig libcluster.ClusterConfig,
 		ctx RolesAddCtx) (paths []path, err error),
-	patchFunc func(config *libcluster.Config, path []string,
-		roleNames []string) (*libcluster.Config, error),
+	patchFunc func(config *libcluster.Config, prt []patchRoleTarget) (*libcluster.Config, error),
```

Comment:
`prt` is something confusing.
```suggestion
	patchFunc func(config *libcluster.Config, target []patchRoleTarget) (*libcluster.Config, error),
```

Reply:
Renamed.